hyp3_gamma/insar/unwrapping_geocoding.py

```diff
@@ -16,6 +17,45 @@
 log = logging.getLogger(__name__)
 
 
+def get_coords(in_mli_par, ref_azlin, ref_rpix, in_dem_par=None):
+    """
+    inputs: mil.par, dempar, reference point  in SAR space (ref_azlin, ref_rpix)
+    returns: coords={"row_s":row_s,"col_s":col_s,"row_m":row_m,"col_m":col_m,"x":x,"y":y,"lat":lat,"lon":lon}
+    """
+    def _coord_lst(cmd):
+        coord_txt = subprocess.run(cmd, capture_output=True, text=True)
+        lst = coord_txt.stdout.split('\n')
+        coord_str = [s for s in lst if "selected" in s]
+        coord_lst = ' '.join(coord_str[0].split()).split(" ")[: -1]
+        coord_lst = [float(s) for s in coord_lst]
+        return coord_lst
+
+    cmd = ['sarpix_coord', in_mli_par, '-']
+    coords = {}
+    if in_dem_par:
+        cmd1 = cmd.copy()
+        cmd1.extend([in_dem_par])
+        cmd1.extend([str(ref_azlin), str(ref_rpix)])
```

Comment:
In general, when sub process commands are made, it's much better to explicitly list the whole command, even if it means repetition

And f-string is easiest to read, but an explicit list is okay too. 

(same for next two places)